src/arith_uint256.cpp

```diff
@@ -146,13 +146,21 @@ double base_uint<BITS>::getdouble() const
 template <unsigned int BITS>
 std::string base_uint<BITS>::GetHex() const
 {
-    return ArithToUint256(*this).GetHex();
+    base_blob<BITS> b;
```

Comment:
I think removing it or moving it is out of scope of this PR, however, just FYI this removes the last non-test use of `ArithToUint256`.

Reply:
Interesting. Didn't notice. I'm happy to reverse this and make `ArithToUint256` use `GetHex`. That would make much more sense.

Reply:
<sorry, scratch my last comment, didn't think it through>

Interesting. Didn't notice. I'm happy to reverse this and make `ArithToUint256` use something from here. That would make much more sense.

Reply:
I think it's fine to leave the implementation of ArithToUint256 as-is. I just meant it can be moved to the test utilities, it's not used in non-test code anymore, but also that that is out of scope for this PR.